### PR TITLE
Update swift Fastfile to new core action names

### PR DESCRIPTION
**Pull request by mpirri**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

We have recently renamed the core _fastlane_ actions and updated Fastfile templates to use those new names. This PR update the Swift Fastfile template to also use the same names.

### CODE REVIEW

File: fastlane/lib/assets/DefaultFastfileTemplate.swift [hunk 1]
```diff
@@ -24,27 +24,26 @@ class Fastfile: LaneFile {
...
     }
 
     func betaLane() {
         desc("Submit a new Beta Build to Apple TestFlight. This will also make sure the profile is up to date")
 
-        match(gitUrl: "gitUrl", appIdentifier: [appIdentifier], username: appleID)
-        // Build your app - more options available
-        gym([[SCHEME]])
-        pilot(username: appleID)
+        // sync_code_signing(gitUrl: "gitUrl", appIdentifier: [appIdentifier], username: appleID)
```
Comment (mpirri) [inline fastlane/lib/assets/DefaultFastfileTemplate.swift]: Commented out `sync_code_signing` (aka `match`) to match the Ruby `Fastfile` template. If this is in error, I'm happy to revert!

File: fastlane/lib/assets/DefaultFastfileTemplate [hunk 2]
```diff
@@ -30,7 +30,7 @@ platform :ios do
   desc "Submit a new Beta Build to Apple TestFlight"
   desc "This will also make sure the profile is up to date"
   lane :beta do
-    # match(type: "appstore") # more information: https://codesigning.guide
+    # sync_code_signing(type: "appstore") # more information: https://codesigning.guide
```
Comment (mpirri) [inline fastlane/lib/assets/DefaultFastfileTemplate]: Whoops missed this one before!

File: fastlane/lib/assets/DefaultFastfileTemplate [hunk 3]
```diff
@@ -42,7 +42,7 @@ platform :ios do
   lane :release do
     # sync_code_signing(type: "appstore")
     capture_screenshots
-    build_app[[SCHEME]] # Build your app - more options available
+    build_app[[SCHEME]] # more options available
```
Comment (mpirri) [inline fastlane/lib/assets/DefaultFastfileTemplate]: `build_app  // build your app`  :point_left: worst comment ever!

File: fastlane/lib/assets/DefaultFastfileTemplate.swift (hunk 1 above)
Comment (taquitos) [reply fastlane/lib/assets/DefaultFastfileTemplate.swift]: need to update the code that substitutes these values in fastlane during initialization `[SCHEME]`

Comment (mpirri) [reply fastlane/lib/assets/DefaultFastfileTemplate.swift]: I'm not sure I follow. The code doesn't match on the action name, so I'd assume there would be no changes to be made there.

Comment (taquitos) [reply fastlane/lib/assets/DefaultFastfileTemplate.swift]: We look for these tokens during initialization and substitute values, check out https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/setup/setup_ios.rb#L301
That way, when you get the fastfile back, it has all the correct values filled in that were specified during `init`.

Comment (taquitos) [reply fastlane/lib/assets/DefaultFastfileTemplate.swift]: Ok, ignore me, I took a closer look and you're got it 💯 correct.

Comment (mpirri) [reply fastlane/lib/assets/DefaultFastfileTemplate.swift]: 👍

**Review by ohayon — APPROVED** [review]

**Pull request merged**